plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorTest.java

```diff
@@ -39,14 +39,14 @@
 import static java.lang.String.format;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertNotNull;
 
 @Test(singleThreaded = true)
-public class TestMongoIntegrationSmokeTest
-        // TODO extend BaseConnectorTest
-        extends AbstractTestIntegrationSmokeTest
+public abstract class BaseMongoConnectorTest
+        extends BaseConnectorTest //changing from AbstractTestDistributedQueries to BaseConnectorTest
```

Comment:
We should also TestMongoDistributedQueries, BaseMongoDistributedQueries, as the BaseConnectorTest extends from AbstractTestDistributedQueries